src/test/groovy/org/ods/doc/gen/pdf/builder/api/DocGenControllerSpec.groovy

```diff
@@ -20,6 +24,8 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 
 @WebMvcTest(DocGenController.class)
 @Slf4j
+@Stepwise
+@DirtiesContext
 class DocGenControllerSpec extends Specification {
```

Comment:
Like with LevaDocServiceTestBase maybe you can create an abstract class named ControllerTestBase and move inside the common annotations between DocGenControllerSpec and LevaDocControllerSpec

Reply:
As we said it is not needed